src/scss/objects/_form-toggle.scss

```diff
@@ -0,0 +1,166 @@
+/**
+ * Form Toggles
+ * --------------------------------
+ * Used for listing page filters
+ *
+ * ================================= */
+
+$formToggle-padding                 : spacing(base) !default;
+$formToggle-border-color            : $grey--lightest;
+$formToggle-border-color-checked    : $grey--lighter;
+$formToggle-border-width            : 1px;
+$formToggle-border-radius           : 8px;
+$formToggle-tick-background         : #04822c;
+$formToggle-background              : $white;
+$formToggle-large-size              : 48px;
+$formToggle-button-color            : $blue;
+$formToggle-button-background       : $blue--offWhite;
+
+.o-formToggle {
+    position: relative;
+    display: inline-block;
+    margin-bottom: spacing(base);
+    border-radius: $formToggle-border-radius;
+    padding: ($formToggle-padding / 2) $formToggle-padding;
+    border: $formToggle-border-width solid $formToggle-border-color;
+
+    &--button {
```

Comment:
Very minor, but I've always avoided using the parent selector as a partial name filler. It makes it really difficult to search for instances in the codebase if you're refactoring, it's easily missed. Happy for you to leave it in but it's worth noting.

e.g 

Rather have `o-formToggle--button` as a selector.